GOVERNANCE.md

```diff
@@ -0,0 +1,195 @@
+# Governance
+
+This document outlines the governance model for Biome. This document includes the contributor model, code review, merging, and the consequences and process for Code of Conduct violations.
+
+## Member Roles
+
+All members must follow the [Code of Conduct](CODE_OF_CONDUCT.md). Consequences for member violations are detailed in [Moderation](#moderation).
+
+### Lead
+
+Leads are the owners of the organisation.
+
+Leads have additional privileges over core contributors. Leads control and maintain sensitive project assets and act as tiebreakers in the event of disagreements.
```

Comment:
An odd number of leaders could avoid the issue?

Reply:
I updated the document, only one lead should step up and make the decision.

@Conaclos that wouldn't work. Being a lead is essentially being a core contributor with more burden (conflict resolution rights, holding sensitive info/assets). We can't force people to join only because we require an odd number.